simphony_ui/ui.py

```diff
@@ -218,7 +227,8 @@ def _run_calc_threaded(self):
                 self.update_progress_bar
             )
         except Exception as e:
-            self.calculation_error_event = str(e)
+            tb = traceback.format_exc().splitlines()
```

Comment:
There's no need to split it and then join it again with '\n' just pass traceback.format_exc() to the tuple.